SQL docs review (#389)
hazelcast/hz-docs: docs/modules/sql/pages/mapping-to-a-file-system.adoc

)
```

|[[options]]_other options_
|none

> |Options to pass to the file system such as <<configuring-authentication-for-remote-file-systems, authentication options ### link doesn't work ###>>.

Reviewer: ```suggestion
|Options to pass to the file system such as <<authentication-for-remote-file-systems, authentication options>>.
```

|[[sharedFileSystem]]`sharedFileSystem`
|`false`

> |`false` - each member will read all files it sees in the directory

Reviewer: ```suggestion
|`false`: Each member reads all files that it sees in the directory.
```

> `true` - assume all members see the same files, they will split the work

Reviewer: ```suggestion
`true`: All members see the same files and split the work.
```

Author: I'd like to keep the verb "assume" here, because you can set the `sharedFileSystem` property to true even for non-shared file system, no error will be reported, but only a part of the files will be processed.

Reviewer: That's important info, so I've added it to the description.

)
```

> All columns have `VARCHAR` type.

Reviewer: ```suggestion
All columns are the `VARCHAR` type.
```
